docs/en/latest/plugins/ip-restriction.md

```diff
@@ -70,6 +70,20 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
+If you want to use a custom message, you can configure it in the plugin section.
```

Comment:
Use a custom message for what?

Reply:
Thanks for the review, I saw https://github.com/apache/apisix/issues/4428, which mentioned ip-restriction plugin support DIY message, was marked `good first issue`, so wanted to try to implement it.  I'll make the changes mentioned above later.

Reply:
Err. Actually I mean we should write it more clear in the documentation, as the user won't have the same context as us.

Reply:
I understand, I will write more clearly later